project4/styles.css

```diff
@@ -0,0 +1,300 @@
+}
+
+img {
+    max-width: 100%;
+    overflow-x: hidden;
+}
+
+
+/* nav li+li {
+    border-left: 1px solid var(--color-white);
+} */
+
+hr {
+    display: block;
+    border-radius: 10px;
+    border-style: none;
+    border: 1.5px solid var(--color-white);
+    margin: 0;
+    margin-left: 30px;
+    text-shadow: none;
+    width: 500px;
+}
+
+section {
+    padding: 100px 30px;
+    /* just for demo purposes */
+}
+
+
+svg path {
+    fill: var(--color-pink);
+}
+
+
+
+#speakers h3 {
+    font-size: 30px;
+}
+
+#exhibitions-learning p {
+    width: 75%;
+}
+
+
+input[type="email"]:focus {
+    border: 3px solid var(--color-pink);
+}
+
+input[type="email"]:required:valid {
+    background-color: var(--color-pink);
+}
+
+.cta-tickets {
+    background-color: var(--color-white);
+    border-radius: 50%;
+    color: var(--color-black);
+    display: block;
+    font-family: bigsmalls-bold, sans-serif;
+    font-size: 11px;
+    line-height: 15px;
+    padding-bottom: 18px;
+    padding-left: 5px;
+    padding-right: 5px;
+    padding-top: 18px;
+    position: absolute;
+    right: 20px;
+    text-align: center;
+    text-decoration: none;
+    top: 30px;
+}
+
+.logo {
+    left: 0;
+    top: 30px;
+    width: 35%;
+}
+
+.theme {
+    color: var(--color-white);
+    font-size: 8px;
+    letter-spacing: 3px;
+    line-height: 1;
+}
+
+
+
+
+.main-title {
+    padding-bottom: 80px;
+    padding-left: 30px;
+    padding-top: 80px;
+}
+
+.main-title p {
+    color: var(--color-pink);
+    font-family: bigsmalls-bold, sans-serif;
+    font-size: 60px;
+    line-height: 25px;
+}
+
+.first {
+    border: 1.5px solid var(--color-pink);
+}
+
+.speaker {
+    border-top-left-radius: 30%;
+    border-bottom-right-radius: 30%;
+    width: 100%;
+
+}
+
+.andrea,
+.sj,
+.arielle {
+    display: block;
+    margin: auto;
+    margin-top: 50px;
+    margin-bottom: 50px;
+    width: 90%;
+}
+
+
+.ticket-type p,
+.ticket-type a {
+    display: inline-block;
+    vertical-align: middle;
+    width: 45%;
+}
+
+.ticket-type a {
+    background-color: var(--color-white);
+    border-radius: 50%;
+    color: var(--color-black);
+    font-family: bigsmalls-bold, sans-serif;
+    font-size: 15px;
+    line-height: 15px;
+    padding-bottom: 55px;
+    padding-top: 55px;
+    text-align: center;
+    text-decoration: none;
+    width: 130px;
+}
+
+.ticket-type p {
+    padding: 30px;
+}
+
+.clearfix::after {
+    clear: both;
+    content: "";
+    display: table;
+}
+
+
+.schedule {
+    text-align: left;
+}
+
+
+.share-list {
+    list-style: none;
+}
+
+.share-list svg {
+    height: 40px;
+    width: 40px;
+}
+
+.share-list a {
+    display: inline-block;
+}
+
+.share-text {
+    margin: auto;
+    padding: 50px;
+    text-align: center;
+}
+
+.share-text h2 {
+    font-size: 40px;
+}
+
+
+
+@media (min-width: 401px) and (max-width: 960px) {
+
+    .andrea,
+    .sj,
+    .arielle {
+        display: inline-block;
+        vertical-align: top;
+        width: 20%;
```

Comment:
Since there are 3 speakers, you can increase this width _and_ if you give them a shared classname (see previous suggestion), you can also add a `margin-left` to space them out a bit in this layout, e.g.:

```
.speaker-container {
        display: inline-block;
        vertical-align: top;
        width: 27%;
}
.speaker-container + .speaker container {
    margin-left: 28px;
}
```
Note, at this size, you're also going to need to make the heading `font-size` smaller to stop the text from overlapping.

All those changes give me this layout at 401px wide:
![Screen Shot 2019-12-12 at 13 37 59](https://user-images.githubusercontent.com/2653980/70739531-0da18880-1ce5-11ea-89eb-6d1287dd63ba.png)